ui/App.svelte

```diff
@@ -73,9 +74,27 @@
   }
 </script>
 
+<style>
+  .error {
+    width: 100vw;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+</style>
+
 <Hotkeys />
 <NotificationFaucet style="margin-top: calc(var(--topbar-height) + 11px)" />
 <Theme />
 <Remote {store} let:data={session} context="session">
   <Router {routes} />
+
+  <div slot="error" class="error">
```

Comment:
This error was bugging me every time I made a change to the session so now it's easier to `clear` it 😄 

![Screen Shot 2020-07-16 at 5 36 40 PM](https://user-images.githubusercontent.com/2712962/87746313-19868900-c7a5-11ea-803c-c68ff46d07bc.png)

Reply:
Tagging: @juliendonck, @brandonhaslegs, so they're aware of it.

Reply:
Wondering if we should keep this, as we already have `yarn nuke` and clearing the session doesn't clear the registry state (can't create the same identity twice).